source/_integrations/zimi.markdown

```diff
@@ -0,0 +1,122 @@
+
+## Supported Devices
+
+The following Zimi devices are supported:
+
+- Zimi Cloud Connect ([links to specifications](https://zimi.life/product/cloud-connect/))
+
+## Unsupported Devices
+
+The following Zimi devices are yet to be supported:
+
+- Zimi Matter Connect ([links to specifications](https://zimi.life/product/matter-connect/))
+
+## Prerequisites
+
+A configured Zimi Cloud Connect and internet connection is needed for this integration to work.
+
+1. Open the app store and install the Zimi app.
+2. Open the Zimi app and configure a Zimi network by adding and naming all Zimi devices.
+3. Open the Zimi app and configure a Zimi Cloud Connect device.
+4. Take a note of the Zimi Cloud Connect IP address and MAC address.
+5. Configure the Zimi integration using standard configuration flow.
+
+{% include integrations/config_flow.md %}
+
+You will be prompted to configure the Zimi Cloud Connect through the Home Assistant interface.
+
+If the Zimi discovery process is successful and there is a single Zimi Cloud Connect, then the integration will be configured without further user input.
+
+If the Zimi discovery process is successful and there are multiple Zimi Cloud Connects present, then the user will be prompted to select the desired Zimi Cloud Connect.
+
+If the Zimi discovery process is unsuccessful (i.e. the Zimi Cloud Connect is not reachable on the local LAN), then the user will be prompted for the following parameters:
+
+{% configuration_basic %}
+host:
+    description: "The IP address of your Zimi Cloud Connect. You can find it via your router admin interface."
+port:
+    description: "The port number used to connect to your Zimi Cloud Connect.   If no port number is entered, the integration will use the default port.   (The default port will be correct in almost all deployment scenarios)"
+{% endconfiguration_basic %}
+
+It is possible to add multiple Zimi Cloud Connect devices.
+
+## Available Entities
+
+The integration will support all Zimi devices.  It should be noted that the naming conventions and default integration types may not be what the user is expecting.
+
+1. Zimi devices that are generic switches will be shown in the UI as a switch and not as a light.   The 'Identify as light for voice control' is not available in the API to pass the necessary information to HA to correctly classify.   For more details of the concept and how to change your device to the correct type after the initial integration, see [Change device type of a switch](/integrations/switch_as_x/).
+2. Zimi devices and names will be mapped per HA guidelines in the table below.   The user may change these names to more friendly names - see [Customizing entities](/docs/configuration/customizing-devices/).
+
+When you add a supported device, the following entities will be created:
+
+| Zimi Product                    | HA Device Name | HA Entities         | HA Default Friendly Name                                         | Supported |
```

Comment:
```suggestion
| Zimi product                    | HA device name | HA entities         | HA default friendly name                                         | Supported |
```
we use [sentence-style capitalization](https://learn.microsoft.com/en-us/style-guide/text-formatting/using-type/use-sentence-style-capitalization) also in headings